Extensions/editable_reblogs.js

```diff
@@ -358,6 +358,13 @@ XKit.extensions.editable_reblogs = new Object({
 					XKit.interface.kitty.set(response.getResponseHeader("X-tumblr-kittens"));
 					XKit.tools.add_function(function() {
 						Tumblr.Events.trigger("postForms:saved");
+						var query_string = window.location.search;
+						if (query_string.indexOf("redirect_to=") !== -1) {
+							var redirect_url = window.location.search.split("redirect_to=")[1];
```

Comment:
this only works if "redirect_to" is the only search parameter. prefer using the jquery param method